config/routes.rb

```diff
@@ -2,16 +2,11 @@
 
 root to: 'accounts#index'
 
-get 'test', to: 'test#jeff'
+# get '/accounts/:id/users', to: 'accounts#users_index', as: 'accounts_users'
 
-get '/accounts', to: 'accounts#index'
+  resources :users, only: [:index, :show]
 
-get '/accounts/:id', to: 'accounts#show', as: 'account'
-
-get '/accounts/:id/users', to: 'accounts#users_index', as: 'accounts_users'
-
-get '/users', to: 'users#index'
-
-get '/users/:id', to: 'users#show', as: 'user'
-
-end
+  resources :accounts, only: [:index, :show] do 
+    resources :users, only: [:index, :show]
+  end 
```

Comment:
+# get '/accounts/:id/users', to: 'accounts#users_index', as: 'accounts_users'

commented this out as well.  I know that this is now pointing towards the users#index.  But not sure what to do in the index action.  if conditional?